src/aat/resources/features/F-105/F-105.feature

```diff
@@ -0,0 +1,201 @@
+@F-105
+Feature: F-105: Add Case-Assigned Users and Roles
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+    And a successful call [to create a token for case creation] as in [Befta_Jurisdiction2_Default_Token_Creation_Data_For_Case_Creation]
```

Comment:
This will create a token before all scenarios - even before those that don't need a case to have been created.

Reply:
Have moved it to each of the relevant scenarios.